src/components/applicant-profile/GrantsList.js

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import { useDispatch, useSelector } from "react-redux";
+import Paper from "@material-ui/core/Paper";
+import List from '@material-ui/core/List';
+import ListItem from '@material-ui/core/ListItem';
+import Button from "@material-ui/core/Button";
+import Grid from "@material-ui/core/Grid";
+import { useStyles } from "./GrantsList.styles";
+
+export default function GrantsList() {
+  const classes = useStyles();
+
+  // const dispatch = useDispatch();
+  // const user = useSelector((state) => state.login.user)
+
+  // useEffect(() => {
+  //   if(user) {
+  //     const { id } = user;
+  //     dispatch(getProfileInfo(id))
+  //   }
+  // },[dispatch, user])
+
+  // const grants = useSelector((state) => state.profileInfo.grants)
+
+  return (
+    <div className={classes.container}>
+      <Grid className={classes.grantslistheader}>
+        <h2>Saved Grants</h2>
+        <div>
+          <Button variant="contained" color="primary" href="#">
+            Add a grant
+          </Button>
+        </div>
+      </Grid>
+      <Paper>
+        <List>
+          <ListItem>
+            <h4>Insert grant name here</h4>
+          </ListItem>
+          <ListItem>
+            <h4>Insert grant name here</h4>
+          </ListItem>
+          <ListItem>
+            <h4>Insert grant name here</h4>
+          </ListItem>
+          {/* {grants.map((grant) => {
```

Comment:
Again, assuming this code is commented out because we don't yet have the endpoint but we will need it once we have endpoints?